### DISCUSSION TITLE
Update documentation to use SQLite

### DISCUSSION THREAD

**Discussion by biapar** [keystonejs/keystone#6544] [Questions] [answered]
Hi,
Could you update the docs to use SQLite?
Thanks.

**Comment by raveling** [comment, marked as answer]
@biapar this is actually landing very soon 🥳 
https://github.com/keystonejs/create-keystone-app/pull/201

**Reply by biapar** [reply]
When?

**Reply by bladey** [reply]
@biapar this landed yesterday - https://github.com/keystonejs/create-keystone-app/pull/202

**Comment by elbotho** [comment]
imho this is solved and can be closed:
https://keystonejs.com/docs/walkthroughs/getting-started-with-create-keystone-app#naming-your-app
